tensorflow/lite/delegates/xnnpack/reshape_tester.cc

```diff
@@ -111,15 +133,25 @@ std::vector<char> ReshapeTester::CreateTfLiteModel() const {
                      OutputShape().size() * sizeof(int32_t))));
   }
 
-  std::vector<flatbuffers::Offset<Tensor>> tensors{{
+  std::vector<flatbuffers::Offset<Tensor>> tensors = {{
```

Comment:
`std::array` would be more appropriate, as the size is known in advance. Also, make it `const`.